lib/app.dart

```diff
@@ -5,12 +5,15 @@ import 'package:ecowave/routes/routes.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:google_fonts/google_fonts.dart';
+import 'package:intl/date_symbol_data_local.dart';
 
 class App extends StatelessWidget {
   const App({super.key});
 
   @override
   Widget build(BuildContext context) {
+    WidgetsFlutterBinding.ensureInitialized();
```

Comment:
15-16 ini seharusnya masuk ke main.dart.
diatas di.init();

correct me kalo ada alasan lain ya

Reply:
Ohh Iyaa bebrapa aku cek banyakan di fungsi mainnya, aku cuma lihat satu referensi aja tadi

Tapi ini aku baca kok tidak disarankan ya
https://meen.academy/flutter/widgetsflutterbinding-ensureinitialized-method-in-flutter/

Itu tadi cuma buat daftarin date ke local id sihh, waktu aku remove jadi error

Reply:
- Kalau menurut aku ```WidgetsFlutterBinding.ensureInitialized();``` itu di fungsi main() sih, sebelum menjalankan runApp().
- Kalau kasus daftarin date ke local, aku kurang faham betul sih. mungkin diambil alternatif terbaik menurut kamu aja.

Reply:
Okei zan udah aku push lagi